src/Amazon.Lambda.Tools/Commands/UpdateFunctionConfigCommand.cs

```diff
@@ -230,8 +230,29 @@ protected async Task ApplyTags(string functionArn)
             }
         }
 
-        protected async Task UpdateConfigAsync(GetFunctionConfigurationResponse existingConfiguration, string dotnetSharedStoreValue)
+        /// <summary>
+        /// Reverts the lambda configuration to an earlier point, which is needed if updating the lambda function code failed.
+        /// </summary>
+        /// <param name="existingConfiguration"></param>
+        /// <exception cref="LambdaToolsException"></exception>
+        protected async Task RevertConfigAsync(GetFunctionConfigurationResponse existingConfiguration)
         {
+            var request = CreateRevertConfigurationRequest(existingConfiguration);
+            await LambdaUtilities.WaitTillFunctionAvailableAsync(Logger, this.LambdaClient, request.FunctionName);
+            this.Logger.WriteLine($"Reverting runtime configuration for function {this.GetStringValueOrDefault(this.FunctionName, LambdaDefinedCommandOptions.ARGUMENT_FUNCTION_NAME, true)}");
+            try
+            {
+                await this.LambdaClient.UpdateFunctionConfigurationAsync(request);
+            }
+            catch (Exception e)
+            {
+                throw new LambdaToolsException($"Error updating configuration for Lambda function: {e.Message}", LambdaToolsException.LambdaErrorCode.LambdaUpdateFunctionConfiguration, e);
```

Comment:
I know this will likely change due to my other comment but this error message should say error reverting configuration not updating configuration.

Reply:
Done